src/Stratis.Bitcoin.LightD/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Api;
+using Stratis.Bitcoin.Features.Apps;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Utilities;
+using Network = Stratis.Bitcoin.Networks.Networks;
+
+namespace Stratis.Bitcoin.LightD
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            try
+            {
+                var nodeSettings = new NodeSettings(networksSelector: Network.Bitcoin, args: args);
+
+                IFullNode node = new FullNodeBuilder()
+                    .UseNodeSettings(nodeSettings)
+                    .UseBlockStore()
+                    .UseLightPowConsensus()
+                    .UseMempool()
```

Comment:
Since we don't have consensus rules enabled is there any point enabling the mempool? We run the risk of relaying invalid transactions from malicious nodes and getting ourselves banned by validating nodes.

Reply:
👍